### ISSUE TITLE
Unable to build after renaming project

### ISSUE THREAD

**Issue by andyfurniss4** [damienbod/Angular2WebpackVisualStudio#72]
I've forked the repo and am now ready to start working with it so I've renamed the project and changed all references of 'Angular2WebpackVisualStudio' to my own project name.

I've deleted the contents of node_modules and run `npm install` and `npm rebuild` to reinstall and rebuild everything.

However I am now getting the following error when doing a dev build and when running the project.

> ERROR in .//css-loader!.//sass-loader!./angularApp/style/app.scss
Module build failed: Error: Missing binding C:\Projects\thematteroffaith\src\TheMatterOfFaith\node_modules\node-sass\vendor\win32-ia32-47\binding.node
Node Sass could not find a binding for your current environment: Windows 32-bit with Node.js 5.x
Found bindings for the following environments:
  Windows 64-bit with Node.js 7.x
This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to build the binding for your current environment.

I'm not sure why it thinks my current environment is 'Windows 32-bit with Node.js 5.x'. I'm running Windows 10 64 bit with node 7.6.0:

![version](https://cloud.githubusercontent.com/assets/16683916/23382757/5624d78c-fd3b-11e6-99d0-fd1e18d99b18.PNG)

And the correct version of node-sass seems to have installed:

![node-sass](https://cloud.githubusercontent.com/assets/16683916/23382828/9ad2abde-fd3b-11e6-9f15-fb762ba01ca3.PNG)

Any ideas as to why it would be looking for 'Windows 32-bit with Node.js 5.x'?

**Comment by robisim74** [comment]
`npm rebuild node-sass` or remove manually `node_modules` and run `npm install` should solve

**Comment by andyfurniss4** [comment]
Sorry, I should have said I tried both of those several times with no luck. I've noticed that `npm run webpack-dev` works in cmd in but the `build dev` task in Visual Studio fails. So Visual Studio fails and I also get the error in the browser console.

**Comment by robisim74** [comment]
Did you try also the @damienbod blog suggestions?

`npm install node-sass -g`

https://blogs.msdn.microsoft.com/webdev/2015/03/19/customize-external-web-tools-in-visual-studio-2015/

**Comment by andyfurniss4** [comment]
`npm install node-sass -g` did not solve the problem but I have found the solution and it's a bit of a facepalm moment for me. The solution was on that blog post you linked but it was the bit at the end about using the node version in my PATH rather than the VS default.

Moved $(PATH) to top of External Web Tools list here: Tools -> Options -> Projects and Solutions -> External Web Tools

I had actually read to do this somewhere before but I'm working on two different PCs at the moment (work and home) and must have forgotten to make the change on my home PC

Problem solved. Thanks for your help Roberto.
